sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/data/CloseableFnDataReceiver.java

```diff
@@ -24,6 +24,8 @@
  * <p>The close method for a {@link CloseableFnDataReceiver} must be idempotent.
  */
 public interface CloseableFnDataReceiver<T> extends FnDataReceiver<T>, AutoCloseable {
+  void flush() throws Exception;
```

Comment:
Please deprecate this method and add a comment about removing time based "push-back" once splitting/checkpointing is available within SDKs.

Reply:
This change broke precommit.

Reply:
Filed https://issues.apache.org/jira/browse/BEAM-5833. A default implementation here should fix it, and I will run spotless, if someone hasn't already addressed it.

Reply:
Ah it is javadoc complaint.

Reply:
I verified that the failing tests were unrelated and also failing at head (and everything passed before this comment was added), but I suppose this was not sufficient. Sorry.